systest/multi-tenancy/basic_test.go

```diff
@@ -47,9 +47,10 @@ var timeout = 5 * time.Second
 // this file can me made common to the other acl tests as well. Needs some refactoring as well.
 func TestAclBasic(t *testing.T) {
 	prepare(t)
-	galaxyToken := testutil.Login(t,
+	galaxyToken, err := testutil.Login(t,
```

Comment:
this is so useful. I have seen tests failing because the token is nil sometimes. Should we check for token being nil?